docs/index.rst

```diff
@@ -4,12 +4,63 @@ Welcome to Cython's Documentation
 
 Also see the `Cython project homepage <https://cython.org/>`_.
 
+Cython is an optimising static compiler for both the Python programming language and
+the extended Cython programming language (based on Pyrex). It makes writing C extensions
+for Python as easy as Python itself.
+
+Quickstart
+-----------
+
+:doc:`src/quickstart/index`
+   How to install Cython compiler package.
+
+:doc:`src/tutorial/index`
+   How to use Cython to speedup your project.
```

Comment:
You are using the verb here, not the noun.
```suggestion
   How to use Cython to speed up your project.
```

Reply:
Thank you again, learning new thing